src/NHSD.BuyingCatalogue.Identity.Api/Infrastructure/ValidationSummaryTagHelper.cs

```diff
@@ -65,9 +70,13 @@ private TagBuilder GetErrorListBuilder()
             var builder = new TagBuilder(TagHelperConstants.UnorderedList);
             builder.AddCssClass(TagHelperConstants.NhsList);
             builder.AddCssClass(TagHelperConstants.NhsValidationSummaryList);
-
+            
             var viewType = ViewContext.ViewData.Model.GetType();
-            viewType.ThrowIfNull();
+
+            if (viewType is null)
+            {
+                throw new InvalidOperationException();
```

Comment:
Previously the function would return `ArgumentNullException`, will this be OK changing the exception to an `InvalidOperationException`?

Reply:
viewType isn't an passed in as an argument so the correct implementation is actually InvalidOperationException